language-plutus-core/src/Language/PlutusCore.hs

```diff
@@ -110,6 +111,9 @@ module Language.PlutusCore
     , EvaluationResult
     -- * Combining programs
     , applyProgram
+    -- * Substitution
+    , substTy
+    , substTerm
```

Comment:
Do we really want to export these rather unsafe functions?

Reply:
They seem okay to me - they're performing a perfectly well-defined operation. I'm just going to end up redefining them if we don't export them.

Reply:
No, I do not mean we shouldn't export them at all, just export them via `Language.PlutusCore.Subst`. I would go as far as name them `naiveSubstTy` and `naiveSubstTerm` or something. It is true they are perfectly well-defined, but they are still dangerous.

Reply:
Okay, I'll just make the `Subst` module exposed, then.